### ISSUE TITLE
where is credentials 2.1.17-SNAPSHOT

### ISSUE THREAD

**Issue by laverite** [hoshsadiq/jenkins-kubernetes-secrets-credentials#1]
Super appreciate this plugin.  I'm new to Jenkins plugins, but I couldn't find a 2.1.17-SNAPSHOT credentials dep published anywhere.  Where can I find it?  Also in general do you plan on continuing with this project or was it mostly just a POC.  Thanks again!

**Comment by hoshsadiq** [comment]
I am planning to continue this plugin, in fact, the plan is to integrate it directly into the Kubernetes plugin. I have had zero time to continue, however, I suggest you do not use this as of yet. As part of using it within my own company, we've had to revert due to SCM polling not working (it doesn't seem to find the secrets during polling for changes, but pulling and running the job works fine). Additionally, I'm planning to overhaul it so that a config and mapping is not necessary either. The plan is that one Kubernetes secret is one Jenkins credentials item.

If you do want to build it and use it, you can build https://github.com/hoshsadiq/credentials-plugin to get 2.1.17-SNAPSHOT into your local maven repository, then build this plugin and it should work.

**Comment by laverite** [comment]
thanks hosh!  if you want any help with any of this, i'm happy to lend a hand.  just let me know.

**Comment by arnaud-deprez** [comment]
Hi,

I don't really understand the initial question as I don't see any release of this plugin but I'm definitely interested in.

Can you tell me the status of it and if you have any issues related to the integration into the kubernetes plugin so we can track the progress and try to help :-). 

I got a similar idea to build such a plugin based on the idea of https://github.com/jenkinsci/openshift-sync-plugin but for secrets to credentials synchronization and then I found this which is directly mentioned in the official https://github.com/jenkinsci/kubernetes-plugin.
So I'm a bit confused, is it already supported or not ? As I cannot see any documentation, I guess it's not yet supported.

Thank you !

**Comment by hoshsadiq** [comment]
It is not yet supported inside the kubernetes plugin. This plugin was more of a PoC, I'm currently trying to integrate this functionality (with some improvements and backwards incompatibilities) into the kubernetes plugin (to me it makes more sense to have it there). Problem is, I haven't had much time to do this. Additionally, as mentioned before, while using this plugin in my current company, it turned out that SCM polling stopped working because of this (required credentials cannot be found). I'm happy to commit and push what I've got so far if you'd like to contribute. It will eventually be merged back into kubernetes plugin once it's done here.

**Comment by hoshsadiq** [comment]
Forgot to mention, I'm also waiting for PR jenkinsci/credentials-plugin#93 to be merged

**Comment by arnaud-deprez** [comment]
Ok, 
I just realized Openshift guys have implemented this in https://github.com/openshift/jenkins-sync-plugin/pull/137

**Comment by hoshsadiq** [comment]
Aah great! That should hopefully make my life easier in implementation for just Kubernetes.

**Comment by arnaud-deprez** [comment]
Yep, or maybe we can kindly ask them if they would agree to put this feature directly in Kubernetes plugin, because there is nothing special to Openshift in this part.
